### PR TITLE
Routing netlify

### PR DESCRIPTION
**Pull request by dilippoudel**

### CODE REVIEW

File: src/components/LogOut/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+const LogOut = () => {
+  return (
+    <>
+      <Button variant="danger">LogOut</Button>
+    </>
+  )
+}
+export default LogOut
```
Comment (Tsaikkari) [inline src/components/LogOut/index.tsx:10]: We already have a logout in the navbar dropdown.

**Review by Tsaikkari — CHANGES_REQUESTED** [review]
You can still work on the profile page; add rows and columns or some other type of structure.

File: src/components/LogOut/index.tsx (hunk 1 above)
Comment (dilippoudel) [reply src/components/LogOut/index.tsx:10]: I had wrote it for the demo in thesis report, forgot to delete :-)

Comment (Tsaikkari) [reply src/components/LogOut/index.tsx:10]: Alright, good idea :)

**Pull request merged**